packages/viz/src/lib/style/helpers/size-bins-style.ts

```diff
@@ -126,7 +126,7 @@ function calculateWithBreaks(
    * @returns radio size.
    */
   const getRadius = (feature: Record<string, any>) => {
-    return getSizeValue(feature);
+    return pixel2meters(getSizeValue(feature), layerStyle);
```

Comment:
In fact, I think we are going to need to use pixels instead of meters to follow the same behavior here and CF, but I would want to talk about it later.

About your specific point, `radiusUnits` are fixed by us, so that conditional will always return the same. Are you thinking about changing `radiusUnits` to use `pixels` in lines and `meters` in points too? If yes, it will make sense for me

Reply:
> About your specific point, `radiusUnits` are fixed by us, so that conditional will always return the same.

Oh, I see, so I should put it as default value instead of fixed.

> Are you thinking about changing `radiusUnits` to use `pixels` in lines and `meters` in points too? If yes, it will make sense for me.

I mean putting the `radiusUnits` as a parameter of the style (with `'pixels'` by default) so by this way the user can specify the points size.

```javascript
// an user who wants size in pixels
const styleInPixels = carto.viz.sizeBinsStyle('availability_365', {
    breaks: [2, 10, 15, 20, 30],
    sizeRange: [2, 10]
});

// an user who wants the size in meters
const styleInMeters = carto.viz.sizeBinsStyle('availability_365', {
    breaks: [50, 100, 200, 500, 1000],
    radiusUnits: 'meters',
    sizeRange: [2, 10]
});
```

Reply:
ah ok, I see. Nice improvement :) I am going to do that way

Reply:
I don't agree. It adds complexity to the user and we don't have it in CF. Let's keep as it was but with the fixed

Reply:
I don't agree. It adds complexity to the user, more work and we don't have it in CF. Let's keep as it's right now.

Reply:
I am going to keep it as it is right now since we have several opinions about it